finance/proposal-calls/cycle4/aperio_general.md

```diff
@@ -23,16 +23,15 @@ This project covers a number of different areas of work around the Astropy proje
 
 #### `astropy.io.fits`
 
-This proposal will fund Tom in his role as a maintainer of `astropy.io.fits`. Our focus will be on continuing to resolve bugs specifically related to the refactoring of the compression module and improve performance, as well as more general maintenance of the sub-package. There are currently over [160 open issues](https://github.com/astropy/astropy/issues?q=is%3Aopen+is%3Aissue+label%3Aio.fits) for the io.fits sub-package alone, and therefore no shortage of work.
+This proposal will fund Robitaille for his work co-maintaining `astropy.io.fits`, continuing funding that was awarded to him to start working on this in 2023. The focus of this work will be to continue to resolve bugs  related to the refactoring of the compression module and improve performance, as well as more general maintenance of the sub-package. There are currently over [160 open issues](https://github.com/astropy/astropy/issues?q=is%3Aopen+is%3Aissue+label%3Aio.fits) for the io.fits sub-package alone, and therefore no shortage of work.
 
-#### `astropy.visualization.wcsaxes`
+#### `astropy.timeseries`
 
-We shall continue maintaining the [astropy.visualization.wcsaxes](https://docs.astropy.org/en/stable/visualization/wcsaxes/) sub-package, which is used to make plots of a wide range of different data with any APE 14-compliant WCS.
-We will work on some of the [open issues](https://github.com/astropy/astropy/issues?q=is%3Aopen+is%3Aissue+label%3Avisualization.wcsaxes) as well as new issues that come up, and will also investigate ways of improving the performance of the plotting.
+Robitaille will continue co-maintaining the [astropy.timeseries](https://docs.astropy.org/en/stable/visualization/wcsaxes/) sub-package as required.
 
-#### `astropy.timeseries`
+#### `astropy.visualization.wcsaxes`
 
-We shall continue maintaining the [astropy.timeseries](https://docs.astropy.org/en/stable/visualization/wcsaxes/) sub-package as required.
+Robitaille and Mumford will continue to maintain the [astropy.visualization.wcsaxes](https://docs.astropy.org/en/stable/visualization/wcsaxes/) sub-package, which is used to make plots of a wide range of different data with any APE 14-compliant WCS. We will work on some of the [open issues](https://github.com/astropy/astropy/issues?q=is%3Aopen+is%3Aissue+label%3Avisualization.wcsaxes) as well as new issues that come up, and will also investigate ways of improving the performance of the plotting.
```

Comment:
I worry this sentence is a little to watery and noncommittal. Should we focus on the performance improvement part?